app/views/ShareListView/Header/SearchBox.tsx

```diff
@@ -60,7 +64,13 @@ const CancelButton = ({ onCancelPress }: { onCancelPress?: () => void }) => {
 	);
 };
 
-const SearchBox = ({ hasCancel, onCancelPress, inputRef, ...props }: ISearchBox): React.ReactElement => {
+const CancelIconButton = (onCancelPress: () => void, theme: TSupportedThemes) => (
```

Comment:
Correct me if I'm wrong, but I think we don't have this feature on Android right now and the focus is only to improve Searchbox without touching this header.
If that's correct, I think we can remove this extra feature.